flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -365,22 +365,45 @@ public static String operatorConfigKey(String key) {
                     .withDescription(
                             "Initial interval of automatic reconcile retries on recoverable errors.");
 
+    @Documentation.Section(SECTION_SYSTEM)
+    public static final ConfigOption<Duration> OPERATOR_RETRY_MAX_INTERVAL =
+            operatorConfig("retry.max.interval")
+                    .durationType()
+                    .defaultValue(Duration.ZERO)
+                    .withDescription(
+                            "Max interval of automatic reconcile retries on recoverable errors, if set to <=0, no limit is applied.");
```

Comment:
How are recoverable errors defined?

Reply:
AFAIK the current default wasn't modified by adding this configuration option, correct?

Reply:
Previously it was infinite, and a recent change actually introduced a 15sec limit accidentally. This is a correction of that. So with this, no change since the last release :)

Reply:
As for the "recoverable error", I just lifted the wording from the other config. It basically means any error that is not handled by the operator logic directly but is "thrown" and therefore should be retried :)